src/pages/workspace/accounting/WorkspaceAccountingPage.tsx

```diff
@@ -0,0 +1,239 @@
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+// import useWaitForNavigation from '@hooks/useWaitForNavigation';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import type {AnchorPosition} from '@styles/index';
+import CONST from '@src/CONST';
+
+function WorkspaceAccountingPage() {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    // const waitForNavigate = useWaitForNavigation();
+    const {isSmallScreenWidth, windowWidth} = useWindowDimensions();
+
+    const [threeDotsMenuPosition, setThreeDotsMenuPosition] = useState<AnchorPosition>({horizontal: 0, vertical: 0});
+    const [policyIsConnectedToAccountingSystem, setPolicyIsConnectedToAccountingSystem] = useState(false);
+    const [isSyncInProgress, setIsSyncInProgress] = useState(false);
+    const [isDisconnectModalOpen, setIsDisconnectModalOpen] = useState(false);
+    const threeDotsMenuContainerRef = useRef<View>(null);
+
+    // fake a QBO connection sync
+    const openQBOsync = useCallback(() => {
+        setIsSyncInProgress(true);
+        setTimeout(() => setIsSyncInProgress(false), 5000);
+        setPolicyIsConnectedToAccountingSystem(true);
+    }, []);
+
+    const connectionsMenuItems: MenuItemProps[] = useMemo(
+        () => [
+            {
+                icon: Expensicons.QBORound,
+                iconType: 'avatar',
+                interactive: false,
+                wrapperStyle: [styles.sectionMenuItemTopDescription],
+                shouldShowRightComponent: true,
+                title: translate('workspace.accounting.qbo'),
+                rightComponent: (
+                    <Button
+                        onPress={openQBOsync}
+                        style={[styles.pl2, styles.justifyContentCenter]}
+                        text={translate('workspace.accounting.setup')}
+                        small
+                    />
+                ),
+            },
+            {
+                icon: Expensicons.XeroRound,
+                iconType: 'avatar',
+                interactive: false,
+                disabled: true,
+                wrapperStyle: [styles.sectionMenuItemTopDescription],
+                shouldShowRightComponent: true,
+                title: translate('workspace.accounting.xero'),
+                rightComponent: (
+                    <Button
+                        style={[styles.pl2, styles.justifyContentCenter]}
+                        text={translate('workspace.accounting.setup')}
+                        small
+                        isDisabled
+                    />
+                ),
+            },
+        ],
+        [openQBOsync, styles.pl2, styles.justifyContentCenter, styles.sectionMenuItemTopDescription, translate],
+    );
+
+    const threeDotsMenuItems: ThreeDotsMenuProps['menuItems'] = useMemo(
```

Comment:
Do we call it `threeDots` in other places? Otherwise I think the correct way to name this is `OverflowMenu`?

Reply:
updated.